client/src/main/java/io/obswebsocket/community/client/WebSocketCloseCode.java

```diff
@@ -22,35 +22,43 @@ public enum WebSocketCloseCode {
   /**
    * A data key is missing but required
    */
-  MissingDataKey(4003),
```

Comment:
I'm not sure from where does that come, but it seems some of those are not CloseCode

Reply:
They are according to the docs, and who am I to disagree :):
https://github.com/obsproject/obs-websocket/blob/master/docs/generated/protocol.md#websocketclosecode

Reply:
You're right, I can't read apparently :D